src/test/java/handlers/CameraHandlerTest.java

```diff
@@ -106,4 +104,26 @@ void testIsActive() {
         CameraHandler ch = new CameraHandler();
         assertFalse(ch.isActive());
     }
+
+    /**
+     * Test setActive method.
+     */
+    @Test
+    void testSetActive() {
+        CameraHandler ch = new CameraHandler();
+        ch.setActive(true);
+        assertTrue(ch.isActive());
+    }
```

Comment:
hier moet je voordat je etactive(true) doet kijken of ie false is

Reply:
Done.